main.star

```diff
@@ -45,6 +47,16 @@ def run(
 
     plan.print("Deploying CDK environment...")
 
+    if deploy_cdk_erigon_node:
```

Comment:
I'll submit a follow-up PR to standardize these parameters, following the approach we took for data availability.